src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -56,14 +60,15 @@ object SharedFileSystem {
    * indirectly through one of its input expressions
    */
   private def localizePathViaSymbolicLink(call: Option[Call], originalPath: Path, executionPath: Path): Try[Unit] = {
-    call.flatMap(_.docker) match {
-      case Some(_) => Failure(new UnsupportedOperationException("Cannot localize with symbolic links with Docker"))
-      case _ => Try(Files.createSymbolicLink(executionPath, originalPath.toAbsolutePath))
-    }
+    if (call.flatMap(_.docker).isDefined)
+      Failure(new UnsupportedOperationException("Cannot localize with symbolic links with Docker"))
+    else if (originalPath.toFile.isDirectory)
+      Failure(new UnsupportedOperationException("Cannot localize directory with symbolic links"))
+    else Try(Files.createSymbolicLink(executionPath, originalPath.toAbsolutePath))
   }
 }
 
-trait SharedFileSystem {
+trait SharedFileSystem extends Backend {
```

Comment:
I don't think so, because SharedFileSystem only implements "localization related functions" of `Backend` but is not a "Backend" in the sense of how we think of what is a backend (Jes, Local, Sge...). I added this because without it IntelliJ doesn't understand that a lot of the functions in there are actually implementations of `Backend` functions. 

I guess the proper way to do this would be to extract all "localization related functions" in the `Backend` trait to a new `Localization` trait. SharedFileSystem could inherit that trait, as well as JesBackend with its own implementation of localization functions.